typescript/submissionEvents.d.ts

```diff
@@ -207,10 +207,17 @@ export type CPHCMSSubmissionEvent = FormEventBase & {
   configuration: PDFConfiguration & {
     /** The content type name for the submission in the CivicPlus HCMS. */
     contentTypeName: string
-    /** An array of element ids to be set as encrypted in the CP HCMS. */
+    /** An array of element ids to be set as encrypted in the CivicPlus HCMS. */
     encryptedElementIds?: string[]
     /** Whether the generated pdf file should be encrypted. (defaults to `false`) */
     encryptPdf?: boolean
+    /** Tags that should be added the content created in the CivicPlus HCMS. */
```

Comment:
```suggestion
    /** Tags that should be added to the content created in the CivicPlus HCMS. */
```

Reply:
*assuming that `to` is what will make sense in terms of the context of what it is used for too